### ISSUE TITLE
blogger/__init__.py

### ISSUE THREAD

**Issue by vijayanandrp** [raghur/easyblogger#10]
```python
from .blogger import EasyBlogger, ContentArgParser
```

It wont raise error if we want to use main.py outside blogger directory.

**Comment by raghur** [comment]
What are you trying to do? Can you provide more details? Are you asking if you can use this as a library?

**Comment by vijayanandrp** [comment]
Yes Raghu, I wanted to use the functionality in [main.py](https://github.com/raghur/easyblogger/blob/master/blogger/main.py) but I got a path error. Thats why I have asked.

**Comment by raghur** [comment]
well you should use it as a module dependency like any other module - install it with pip and then import the EasyBlogger class into your app/script.

**Comment by vijayanandrp** [comment]
First of all I have tried everything using Pycharm Community IDE.
I  wanted to use `main.py` as a CLI tool in my local machine. When I copied the code from `main.py` to user space. Then tried importing `import ContentArgParser` as IDE showed as unresolved reference.  

When tried to fix this, I have added below code in `C:\Program Files (x86)\Python35-32\Lib\site-packages\blogger\__init__.py`
```python
from .blogger import EasyBlogger, ContentArgParser
```
It worked for me. Anyways  thanks for the great tool.
